src/providers/FileSystemProvider/FileSystemProvider.ts

```diff
@@ -24,7 +24,8 @@ export type Entry = File | Directory;
 export function generateFileContent(fileName: string, sourceContent: Buffer): { content: string[]; enc: boolean } {
   const sourceLines = sourceContent.toString().split("\n");
   const fileExt = fileName.split(".").pop().toLowerCase();
-  if (fileExt === "cls") {
+  const csp = fileName.startsWith("/");
```

Comment:
Why are these modifications necessary? Can classes and routines be stored in a web application folder? That seems weird and a bad idea.